test/e2e-tests.sh

```diff
@@ -28,7 +28,7 @@ SKIP_GO_E2E_TESTS=${SKIP_GO_E2E_TESTS:="false"}
 E2E_GO_TEST_TIMEOUT=${E2E_GO_TEST_TIMEOUT:="20m"}
 RUN_FEATUREFLAG_TESTS=${RUN_FEATUREFLAG_TESTS:="false"}
 RESULTS_FROM=${RESULTS_FROM:-termination-message}
-ENABLE_STEP_ACTIONS=${ENABLE_STEP_ACTIONS:="false"}
+ENABLE_STEP_ACTIONS=${ENABLE_STEP_ACTIONS:="true"}
```

Comment:
Since `StepActions` is GA after this PR, we don't need to enable it in the test script. The flag should go away